py/vtdb/tablet.py

```diff
@@ -302,11 +336,15 @@ def _stream_execute(self, sql, bind_variables):
   # the conversions will need to be passed back to _stream_next
   # (that way we avoid using a member variable here for such a corner case)
   def _stream_execute2(self, sql, bind_variables):
```

Comment:
After your changes, _stream_execute and _stream_execute2 should be identical (since there are no backwards compatibility concerns now). You should remove all traces of _stream_execute2.